components/automate-ui/src/app/pages/client-runs/client-runs.component.ts

```diff
@@ -164,7 +164,7 @@ export class ClientRunsComponent implements OnInit, OnDestroy {
   nodeCounts$: Observable<NodeCount>;
 
   // Does the request have the correct permissions
-  permissionDenied$: Observable<boolean>;
+  permissionDenied$: Observable<boolean>; // not currently used
```

Comment:
Why keep it then?

Reply:
Sounded like it might be coming back soon, though rendered slightly differently. 🤷